Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -180,6 +191,8 @@ class LibraryViewController: UIViewController {
             } else {
                 navigationController?.setToolbarHidden(false, animated: true)
             }
+        case .history(_):
```

Comment:
I think this can work too?
```suggestion
        case .history:
```